### PR TITLE
feat(graphic): Add schema for GrahpicContainer

### PR DESCRIPTION
**Pull request by chriswmackey**

### CODE REVIEW

File: ladybug_display_schema/graphic.py [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+    )
+
+    name: str = Field(
+        ...,
+        description='Text to indicate how the data type displays. This can be more '
+        'specific than the data_type.'
+    )
+
+
+class GraphicContainer(BaseModel):
```
Comment (mostaphaRoudsari) [inline ladybug_display_schema/graphic.py:233]: I think we need two validators here:

1. Check the match between values and geometry.
2. Check that either geometry or (min and max) are being provided.

File: ladybug_display_schema/graphic.py [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+    volume = 'Volume'
+    volume_flow_rate = 'VolumeFlowRate'
+    volume_flow_rate_intensity = 'VolumeFlowRateIntensity'
+    wet_bulb_temperature = 'WetBulbTemperature'
+    wind_direction = 'WindDirection'
+    wind_speed = 'WindSpeed'
+    zenith_luminance = 'ZenithLuminance'
+
+
+class DataTypeBase(BaseModel):
```
Comment (mostaphaRoudsari) [inline ladybug_display_schema/graphic.py:214]: Why not calling this DataType? I don't see any subclasses of this base class.

**Review by mostaphaRoudsari — APPROVED** [review]
Looks good to me. There are a couple of minor typos in docstring.

I also suggest to add some validation to `GraphicContainer`.

File: ladybug_display_schema/graphic.py (hunk 2 above)
Comment (chriswmackey) [reply ladybug_display_schema/graphic.py:214]: This is a good point and I guess this was just a hold over from when we first put together the DataType module some ~3 years ago. I think I can change it from DataTypeBase to DataType and everything should still work.

**Pull request merged**